packages/apollo-voyager-conflicts/README.md

```diff
@@ -0,0 +1,4 @@
+# `@aerogear/apollo-voyager-conflicts`
```

Comment:
-1 for name. We could something that will reflect both offline state and conflicts resolution.
Any suggestion?